### DISCUSSION TITLE
Registre d'extension

### DISCUSSION THREAD

**Discussion by FrancisLabetoulle** [brackets-cont/brackets#32] [Q&A]
Bonjour
Je n'arrive pas à ouvrir le registre d'extension de Brackets.
Il semblerait que le problème  vient du fichier: (sur mac )
Applications/Brackets.app/Contents/www/utils/ExtensionUtils.js (ligne 269) selon 👍 Default extension has been desabled on startup.
Ce fichier est en lecture seule (apparemment).
Comment résoudre ce problème.
merci d'avance .
FranLab

**Comment by abose** [comment]
Hi @FrancisLabetoulle Apologies for the inconvenience, the issues will be fixed by the November release of Brackets.

NB: The issues happened as Adobe switched off its servers hosting various brackets-related services. We are in the process of transitioning brackets into a community-maintained project. We are actively working on fixes.

More details can be found here: https://github.com/brackets-cont/brackets/issues/34

**Comment by FrancisLabetoulle** [comment]
Grand merci

J'attends la version de Novembre avec espoir.
Amicalement
Francis Labetoulle
